### PR TITLE
44-2-top-bar-group5

### LINKED ISSUE #44: [Feature] Top Bar
### Description

Top bar of the GUI. Copy the drop down menus and options from the old GUI

### PR DESCRIPTION
**Pull request by christian140398**
Closes #44 #2 

### Changes/Additions:

- Added Topbar
- Added All navigations
- Missing functionality for buttons, Open project is implemented as example, but does nothing yet

### CODE REVIEW

**Review by KamyaPA — DISMISSED** [review]
LGTM

File: yarn.lock [hunk 1]
```diff
@@ -1739,7 +1739,7 @@ svelte-check@^3.5.2:
     postcss "^8.4.29"
     postcss-scss "^4.0.8"
 
-svelte-google-materialdesign-icons@^0.8.2:
+svelte-google-materialdesign-icons@^0.8.1:
```
Comment (NybyDK) [inline yarn.lock:1742]: no downgrade pls :(

File: src/lib/components/topBar/TopBar.svelte [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+			name="Autoscaling"
+			on:click={() => {
+				console.log("Autoscaling");
+			}}
+		/>
+		<DropDownBtn
+			name="Scalling"
+			on:click={() => {
+				console.log("Scalling");
+			}}
```
Comment (NybyDK) [inline src/lib/components/topBar/TopBar.svelte:116]:
```suggestion
			name="Scaling"
			on:click={() => {
				console.log("Scaling");
			}}
```

**Review by NybyDK — CHANGES_REQUESTED** [review]
besides this i am ready to approve

**Review by Jesper276 — CHANGES_REQUESTED** [review]
image

can you add icons?

**Review by Jesper276 — CHANGES_REQUESTED** [review]
image
can the submenu functionality also be added?!?

**Review by NybyDK — DISMISSED** [review]

File: src/lib/components/topBar/TopBar.svelte [hunk 3]
```diff
@@ -0,0 +1,204 @@
+<script lang="ts">
+	import TopBarBtn from "$lib/components/topBar/TopBarBtn.svelte";
+	import DropDownBtn from "$lib/components/topBar/DropDownBtn.svelte";
+	import { load } from "$lib/classes/project";
+	import * as Icon from "svelte-google-materialdesign-icons";
```
Comment (NybyDK) [inline src/lib/components/topBar/TopBar.svelte:5]: according to our svelte expert, this is not good for the bundling, instead, we should only import the ones we use

Comment (NybyDK) [reply src/lib/components/topBar/TopBar.svelte:5]: i only meant to highlight line 5 xd

**Review by Jesper276 — APPROVED** [review]
Lgtm

**Review by KamyaPA — APPROVED** [review]
LGTM

**Pull request merged**